pages/index.tsx

```diff
@@ -10,11 +11,12 @@ export default function Home() {
       </Head>
 
       <main>
-        <Logo></Logo>
-        <div></div>
+        <Link href={`/traveling/`}>
+          <a>
+            <Logo></Logo>
```

Comment:
Da das Logo eine eigene Komponente darstellt und hier keine child-elemente haben wird, würde ich verkürzt schreiben (selfclosing) 😊 Das macht den Code auf lange Sicht auch etwas übersichtlicher
```suggestion
            <Logo/>
```